shared/models/errors/advanced.yml

```diff
@@ -0,0 +1,33 @@
+type: string
+enum:
+  - address_length_exceeds_limit
+  - bank_account_already_verified
+  - bank_error
+  - custom_envelope_inventory_depleted
+  - deleted_bank_account
+  - failed_deliverability_strictness
+  - file_pages_below_min
+  - file_pages_exceed_max
+  - file_size_exceeds_limit
+  - foreign_return_address
+  - inconsistent_page_dimensions
+  - invalid_bank_account
+  - invalid_bank_account_verification
+  - invalid_check_international
+  - invalid_country_covid
+  - invalid_file
+  - invalid_file_dimensions
+  - invalid_file_download_time
+  - invalid_file_url
+  - invalid_image_dpi
+  - invalid_international_feature
+  - invalid_perforation_return_envelope
+  - invalid_template_html
+  - merge_variable_required
+  - merge_variable_whitespace
+  - payment_method_unverified
+  - pdf_encrypted
+  - special_characters_restricted
+  - unembedded_fonts
+description: >
```

Comment:
Where are the line item descriptions for the individual enum error values?
For now, let's include the status code in the enum description, so that the resulting documentation looks roughly the same as it does in the current docs.